VK/VK/Presentation/Authentification/ViewControllers/AuthentificationViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  AuthentificationViewController.swift
+//  VK
+//
+//  Created by Алена Панченко on 22.11.2022.
+//
+
+import UIKit
+
+import WebKit
+
+/// Экран аутентификации
+final class AuthentificationViewController: UIViewController {
+    
+    // MARK: - Private Constants
+    private enum Constants {
+        static let urlComponentsSchemeName = "https"
+        static let urlComponentsHostName = "oauth.vk.com"
+        static let urlComponentsPathName = "/authorize"
+        static let queryItemsIdName = "client_id"
+        static let queryItemsURLRedirectName = "redirect_uri"
+        static let queryItemsDisplayName = "display"
+        static let queryItemsScopeName = "scope"
+        static let queryItemsResponseTypeName = "response_type"
+        static let queryItemsVersionName = "v"
+        static let queryItemsRevokeName = "revoke"
+        static let queryItemsURLRedirectValue = "https://oauth.vk.com/blank.html"
+        static let queryItemsDisplayValue = "mobile"
+        static let queryItemsScopeValue = "262150"
+        static let queryItemsResponseTypeValue = "token"
+        static let queryItemsVersionValue = "5.131"
+        static let queryItemsRevokeValue = "0"
+        static let urlPath = "/blank.html"
+        static let separationOfComponents = "&"
+        static let equalSign = "="
+        static let accessTokenName = "access_token"
+        static let storyBoardName = "Main"
+        static let tabBarControllerIdentifier = "TabBar"
+        
+    }
+    // MARK: - Private Outlets
+
+    @IBOutlet var wKWebView: WKWebView! { didSet {
```

Comment:
```suggestion
    @IBOutlet private var wKWebView: WKWebView! { didSet {
```
и кажется уехали скобки куда-то